addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -286,11 +286,8 @@ def _l10n_es_edi_get_invoices_info(self, invoices):
                         **partner_info,
                         'NombreRazon': com_partner.name[:120],
                     }
-
-                if not com_partner.country_id or com_partner.country_id.code in eu_country_codes:
-                    invoice_node['ClaveRegimenEspecialOTrascendencia'] = '01'
-                else:
-                    invoice_node['ClaveRegimenEspecialOTrascendencia'] = '02'
+                export_exempts = invoice.invoice_line_ids.tax_ids.filtered(lambda t: t.l10n_es_exempt_reason == 'E2')
```

Comment:
So if one of the taxes is exempt, all the taxes must be exempt?
Is that true?

Reply:
@william-andre If you use exemption E2, it is because you export.  We have a test where we add also a service that is accessory and then it should stay 02.  (even if that line is not E2)

Asking the virtual assistant he also says that accessory products should be informed as 02.  (https://www2.agenciatributaria.gob.es/wlpl/AVAC-CALC/AsistenteSII )